packages/reselect-utils/src/createPropSelector.ts

```diff
@@ -6,7 +6,9 @@ import {
 
 const propSelectorSymbol = Symbol.for('PropSelector');
 
-export const isPropSelector = (selector: unknown) =>
+export const isPropSelector = (
+  selector: unknown,
+): selector is { path: (string | number)[] } =>
```

Comment:
I propose to create `type Path = string[]` because `Object.keys` always returns strings.